notify/options.go

```diff
@@ -36,18 +36,31 @@ func WithSender(sender string) Option {
 	}
 }
 
-// WithRecipient sets the recipient email address.
+// WithRecipient sets a single recipient email address.
 func WithRecipient(recipient string) Option {
```

Comment:
Could it make sense to make the WithRecipient option append a recipient rather than set that as the only recipient?

Reply:
or maybe check that it isn't trying to override a previous recipient?

Reply:
Unlike filters, we need a way to override a recipient. The new WithRecipients can be used for multiple recipient.